modules/ROOT/pages/7.1-release-notes.adoc

```diff
@@ -109,10 +109,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} 7.1 also includes the following improvement<s>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== In the custom view, the scrollbar of the container were not visible if its height was greater than the editor.
```

Comment:
@tiny-ben-tran Can you confirm the title for this, your `.changie.yml` entry seems like it was cut-off in your PR.